### DISCUSSION TITLE
The parameter 'grid_limits' of function' grid_from_radars' affects the  latitude in the grid

### DISCUSSION THREAD

**Discussion by nomatter08** [ARM-DOE/pyart#1288] [Q&A] [answered]
when i set grid_limits=((0.,20000), (-1000000., 1000000.), (-1000000., 1000000.))，the result looks like move southward

![1](https://user-images.githubusercontent.com/90033942/131981886-63c1567b-da5b-4e96-9fd0-644a8c303ad6.png)

when i set grid_limits=((0.,20000), (-1000000., 1000000.), (-500000., 500000.))，the result looks better and the  location of radar is closer to the real situation

![2](https://user-images.githubusercontent.com/90033942/131983257-ccc70107-3d3b-4751-9021-e89dc3d38771.png)

I tried to change a lot of parameters and it seems that only this  parameter will change  the latitude of grid

**Comment by rcjackson** [comment, marked as answer]
What are the number of points you are using for each direction in the grid in both examples? If you have fewer points spanning a longer distance, you're going to get a less accurate location since the grid will not cover all of the needed points.
